package.json

```diff
@@ -15,9 +15,6 @@
     "postinstall": "node scripts/installScript.js",
     "cleanup": "node scripts/cleanup.js"
   },
-  "cpu": [
-    "!arm64"
```

Comment:
Looks like you may need to update the README [here](https://github.com/Govind-gupta75/devtools-cli/blob/cli-fixes/packages/uxp-devtools-cli/README.md?plain=1#L64) to reflect the change in arm support?

Reply:
@mikemahony yes, I am working on fixing these.